pyClient/README.md

```diff
@@ -45,11 +55,211 @@ and fails to fetch it (or fails to fetch the right version).
 
 ```console
 # Download the solidity compiler to compile the contracts
-$ wget https://github.com/ethereum/solidity/releases/download/[solc-version]/[solc-for-your-distribution] -O $ZETH/pyClient/zeth-devenv/lib/[python-version]/site-packages/solcx/bin/solc-[solc-version]
+$ wget https://github.com/ethereum/solidity/releases/download/[solc-version]/[solc-for-your-distribution] \
+    -O $ZETH/pyClient/zeth-devenv/lib/[python-version]/site-packages/solcx/bin/solc-[solc-version]
 $ chmod +x $ZETH/pyClient/zeth-devenv/lib/[python-version]/site-packages/solcx/bin/solc-[solc-version]
 ```
 
-To run this command you need to replace the solidity version (denoted by
-`[solc-version]`), the python version (denoted `[python-version]`), and binary
-file (denoted by `[solc-for-your-distribution]`) by your system specific pieces
-of information.
+To run this command, replace the solidity version (denoted by `[solc-version]`),
+the python version (denoted by `[python-version]`), and binary file (denoted by
+`[solc-for-your-distribution]`) by your system specific information.
+
+# The `zeth` command line interface
+
+The `zeth` command exposes Zeth operations via a command line interface.  A
+brief description is given in this section.  More details are available via
+`zeth --help`, and example usage can be seen in the [pyclient test
+script](../scripts/test_zeth_cli).
+
+## Environment
+
+Depending on the operation being performed, the `zeth` client must:
+- interact with an Ethereum RPC host
+- interact with the deployed Zeth contracts
+- request proofs and keys from the `prover_server`, and
```

Comment:
Please specify that it is the Verification key (otherwise it can get confusing)

Reply:
Changed to 'proof verification key'